packages/core/src/main/conversation/ConversationService/ConversationService.ts

```diff
@@ -1148,7 +1148,15 @@ export class ConversationService {
       throw new Error('You need to pass self user qualified id in order to create an MLS conversation');
     }
 
-    await this.mlsService.createConversation(groupIdDecodedFromBase64);
+    const mlsKeys = (await this.apiClient.api.client.getPublicKeys()).removal;
+    const mlsKeyBytes = Object.values(mlsKeys).map((key: string) => Decoder.fromBase64(key).asBytes);
+    const config: ConversationConfiguration = {
+      externalSenders: mlsKeyBytes,
+      ciphersuite: 1,
```

Comment:
I think we have an enum in core-crypto for ciphersuite identifiers, don't we?

Reply:
when i used the enum before it was sending back the name of the cipher and not the number, but i will check again now.

Reply:
![Screenshot 2022-09-22 at 17 12 50](https://user-images.githubusercontent.com/37285713/191787199-7d06c5f8-1540-433b-82aa-962a8502827f.png)
unfortunately the enum from CoreCrypto is currently undefined in value, while the config needs a numerical value.

Reply:
Can you try with the latest core-crypto (if you rebase to the latest main branch) maybe. I think there was a change around this in the 0.5.0 (in case you are not using this one of course)

Reply:
this has just been merged with the main branch

Reply:
will follow up with core-crypto folks but we are blocked here for now i believe

Reply:
this is broken still, merging for now with a TODO not to update when its fixed